### PR TITLE
hide source on config generation

### PR DESCRIPTION
**Pull request by amanenk**
fixes cloudquery/cloudquery-issues#176

### CODE REVIEW

File: pkg/client/client_test.go [hunk 1]
```diff
@@ -609,6 +613,7 @@ func Test_collectProviderVersions(t *testing.T) {
 }
 
 func Test_CheckForProviderUpdates(t *testing.T) {
+	soruce := "cloudquery"
```
Comment (roneli) [inline pkg/client/client_test.go:616]: Typo

File: pkg/client/client_test.go [hunk 2]
```diff
@@ -467,6 +469,8 @@ func setupTestPlugin(t *testing.T) context.CancelFunc {
 	defer watcher.Close()
 
 	go provider.ServeTestPlugin(debugCtx)
+	// sleep to prevent ParseReattachProviders from reading empty file
+	time.Sleep(time.Millisecond * 50)
```
Comment (roneli) [inline pkg/client/client_test.go:473]: This feels flaky to me, would use a file watcher I think such a check already exists in our tests

**Review by roneli — CHANGES_REQUESTED** [review]
Looks good, two minor issues and we can merge

File: pkg/client/client_test.go (hunk 2 above)
Comment (amanenk) [reply pkg/client/client_test.go]: it seems that error was related to github request limits. Now it works without these lines

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**